py_mmd_tools/yaml_to_adoc.py

```diff
@@ -70,6 +70,9 @@ def set_attribute(mmd_field, val, convention, attributes, req='not_required'):
         raise ValueError('val input must be dict')
     if convention not in val.keys():
         return None
+    if mmd_field.split('.')[0] == 'not_mmd':
+        mmd_field = 'None'
```

Comment:
Have you tried to run the script to create the adoc-file? I guess the table will show the text "None", then? Maybe better with
```suggestion
        mmd_field = ""
```
?

Reply:
Yes exactly, currently it shows the text "None", I'll change this to ""